src/Backend/Core/Engine/TemplateModifiers.php

```diff
@@ -272,4 +272,31 @@ public static function truncate($var = null, $length, $useHellip = true, $closes
             return \SpoonFilter::htmlspecialchars($var, ENT_QUOTES);
         }
     }
+
+    /**
+     * Shows a v or x to indicate the boolean state (Y|N, j|n, true|false)
+     *
+     * @param string|bool $status
+     * @param bool        $reverse show the opposite of the status
+     * @return string
+     */
+    public static function showBool($status, $reverse = false)
+    {
+        if (!$reverse) {
+            if ($status === 'Y' || $status === 'y' || $status === 1 || $status === '1' || $status === true) {
+                return '<strong style="color:green">&#10003;</strong>';
+            } elseif ($status === 'N' || $status === 'n' || $status === 0 || $status === '0' || $status === false) {
```

Comment:
Why would you use an else, if you already have a return statement in your previous statement?